### PR TITLE
Fixes wormhole generation

### PR DESCRIPTION
**Pull request by Corsaka**
## About The Pull Request
Changes wormhole generation to allow wormholes between two random systems, rather than one random system and Risa Station.

## Why It's Good For The Game
Allows wormholes to spawn somewhere other than between Risa and a random location.

## Changelog
:cl:
fix: Wormhole generation
/:cl:

### CODE REVIEW

File: nsv13/code/controllers/subsystem/starsystem.dm [hunk 1]
```diff
@@ -266,18 +266,16 @@ SUBSYSTEM_DEF(star_system)
 	addtimer(CALLBACK(src, .proc/generate_anomaly), 15 SECONDS)
 
 /datum/star_system/proc/create_wormhole()
-	for(var/datum/star_system/S in SSstar_system.systems)
-		if(LAZYFIND(adjacency_list, S)) //We're already linked to that one. Skip it.
-			continue
+	var/datum/star_system/S = SSstar_system.systems[(rand(26))] //Pick a random system to put the wormhole in.
```
Comment (Kmc2000) [inline nsv13/code/controllers/subsystem/starsystem.dm:269]:
```suggestion
	var/datum/star_system/S = SSstar_system.systems[(rand(SS_star_systems.systems.len))] //Pick a random system to put the wormhole in.

```

Comment (Corsaka) [reply nsv13/code/controllers/subsystem/starsystem.dm:269]: No, that includes Abassi, Dolos, and DATA EXPUNGED, which we don't want.

Comment (Corsaka) [reply nsv13/code/controllers/subsystem/starsystem.dm:269]: Unless we do want that.

Comment (Kmc2000) [reply nsv13/code/controllers/subsystem/starsystem.dm:269]: UNLESS????

Comment (Corsaka) [reply nsv13/code/controllers/subsystem/starsystem.dm:269]: Helpful answer.

**Comment by Corsaka** [comment]
The only problem I've noticed in testing is that the two-way code doesn't seem to work, though without knowledge of what LAZYFIND actually does I don't think I can fix that,

**Comment by Corsaka** [comment]
Gonna have to move back to draft until I can figure out why two-way doesn't work. Also coding in a few other fixes related to blacksite.

**Comment by Vasily2013** [comment]
Please ensure that if there are any issues related to this that they are linked.

**Pull request merged**